js/templates/modals/listingDetail/listing.html

```diff
@@ -2,8 +2,19 @@
 
 <div class="modalContentInner">
   <div class="listingControls flex">
+    <div class="jsModalClose contentBox clrP clrBr clrSh3" style="height: 35px; line-height: 35px; padding: 0 8px; cursor: pointer">
+      <div class="userIcon floL discTn clrBr2 clrSh1"
```

Comment:
The avatar is aligned strangely, the vertical flex class is probably what you want here. 

![image](https://cloud.githubusercontent.com/assets/1584275/21531809/dfc753dc-cd18-11e6-8358-fa0ec431d970.png)